src/test/java/com/openlattice/rehearsal/data/DataControllerTest.kt

```diff
@@ -52,6 +52,7 @@ import java.util.*
 
 private val numberOfEntries = 10
 private val random = Random()
+private val OL_ID_FQN = FullQualifiedName("openlattice", "@id")
```

Comment:
Isn't this available as a static request?

Reply:
I can't find it, in SearchService, we also use string literals